war/.gitignore

```diff
@@ -1,2 +1,10 @@
-work
+work
```

Comment:
@KostyaSha Feel free to start a dev list thread on the topic. The pattern has been in play for some time now.